AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -49,16 +49,6 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutAlignItems) {
   ASStackLayoutAlignItemsStretch,
 };
 
-typedef struct {
-  /** Specifies the direction children are stacked in. */
-  ASStackLayoutDirection direction;
-  /** The amount of space between each child. */
-  CGFloat spacing;
-  /** How children are aligned if there are no flexible children. */
-  ASStackLayoutJustifyContent justifyContent;
-  /** Orientation of children along cross axis */
-  ASStackLayoutAlignItems alignItems;
-} ASStackLayoutSpecStyle;
```

Comment:
Yes!  Kill it with fire! (ASStackLayoutSpecStyle, hate these kind of struct-requiring initializer patterns)